doc/dev/getting-started.rst

```diff
@@ -203,42 +202,32 @@ Claiming an issue and starting to develop
    changed. A file in the 'untracked' section are files that haven't existed
    previously in the repository but git has noticed.
 
-   To commit changes you have to 'stage' them--this is done by issuing the
+   To commit changes you have to 'stage' them—this is done by issuing the
    following command::
 
       git add path/to/file
 
-   If you have a large quantity of changes and you don't want to add each file
-   manually you can do ``git add --patch`` which will display each set of
-   changes to you before staging them for commit.
-
    Once you have staged your changes, it's time to make a commit::
 
-      git commit
+      git commit -m 'Here you provide a brief description of your changes'
```

Comment:
I prefer recommending the text editor route. It seems to encourage people ever so slightly more to write a short subject line and then a few lines of why in the commit message instead of a single line comment. But #bikeshedding